src/main/scala/me/rexim/issuestant/Main.scala

```diff
@@ -1,5 +1,57 @@
 package me.rexim.issuestant
 
-object Main extends App {
-  println("Hello, Issuestant!")
+import io.circe._
+import io.circe.generic.auto._
+import io.circe.parser._
+import io.circe.syntax._
+
+import scala.io._
+
+case class IssueWithParent(parentNumber: Option[Int], issue: Issue);
+
+object Main {
+  def extractParent(issue: Issue): IssueWithParent = {
+    val splitCookie = "(?i)split from #([0-9]+)".r
```

Comment:
I'd use something like `(?i)split +from + #([0-9]+)`here because multiple spaces is a common typo and it's harder to spot than ordinary misspelling.